src/content/concepts/mode.md

```diff
@@ -26,12 +26,13 @@ or pass it as a [CLI](/api/cli/) argument:
 webpack --mode=production
 ```
 
-The following string values are supported:
+If unset, webpack sets `production` as a default for mode and warns to let you know about this behavior. The supported values for mode are:
```

Comment:
Just to be grammatically correct and more reader friendly, shouldn't we say `If not set/provided,  webpack sets/uses production as default value for mode`. Otherwise LGTM